kythe/go/extractors/config/runextractor/constants/constants.go

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 The Kythe Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package constants defines relevant constants common to multiple extractors.
+package constants
+
+import ()
```

Comment:
dd

Reply:
Does your editor add these? I've seen this a couple times now, and I'm not sure where they come from.

Reply:
Yea auto drop in by the editor.